source/fundamentals/crud/read-operations/count.txt

```diff
@@ -0,0 +1,174 @@
+===============
+Count Documents
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to count the number of documents
+in your collection.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the example in this guide, load the sample data into the
+``ratings`` collection of the ``tea`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/countAndEstimate.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+.. include:: /includes/fundamentals/automatic-db-coll-creation.rst
+
+Each document contains a rating for a type of tea that corresponds to
+the ``type`` and ``rating`` fields.
+
+Accurate Count
+--------------
+
+To count the number of documents that match your query filter, use the
+``CountDocuments()`` function.
```

Comment:
[question/ optional suggestion] "Accurate Count" was a little confusing to me until I saw "Estimated Count" - do you think it's worth adding a sentence along these lines?
```suggestion
``CountDocuments()`` function. To quickly estimate the number of documents that match your query filter instead, see :ref:`<estimated-count>`
```
(and a similar one under estimate - "To get an accurate count of the number of documents..."

I think it could help contextualize the two different Counts quickly, but I can also see keeping each section focused. Just a suggestion, deferring to you.